src/main/kotlin/uk/gov/justice/digital/hmpps/personrecord/service/person/UnmergeService.kt

```diff
@@ -40,7 +41,7 @@ class UnmergeService(
   }
 
   private fun retrieveUnmergedPerson(unmergeEvent: UnmergeEvent, unmergedPersonCallback: () -> PersonEntity?): PersonEntity =
-    personService.processMessage(unmergeEvent.unmergedRecord, unmergeEvent.event) {
+    createUpdateService.processMessage(unmergeEvent.unmergedRecord, unmergeEvent.event) {
```

Comment:
Going to refactor this in subsequent PR as in current implementation message could be retried upto 25 times 😱 . Due to retrying on `UnmergeService` and the old `PersonService`